src/test/resources/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs/testAlleleSpecificAnnotations.CombineGVCF.expected.g.vcf

```diff
@@ -54,25 +54,25 @@
-20	10433328	.	A	T,ATATAT	2087.37	.	AC=3,1;AF=0.750,0.250;AN=4;AS_BaseQRankSum=nul,nul;AS_FS=0.000,0.000;AS_MQ=43.36,29.00;AS_MQRankSum=nul,nul;AS_QD=25.36,28.73;AS_ReadPosRankSum=nul,nul;AS_SOR=0.693,1.179;DP=66;ExcessHet=3.0103;FS=0.000;MLEAC=3,1;MLEAF=0.750,0.250;MQ=41.73;QD=30.97;RAW_MQ=114928.00;SOR=0.760	GT:AD:GQ:PL	1/2:0,6,3:99:919,177,246,420,0,500	1/1:0,20,0:74:1191,74,0,1132,93,1376
+20	10433328	.	A	T,ATATAT	2087.37	.	AC=3,1;AF=0.750,0.250;AN=4;AS_BaseQRankSum=.,.;AS_FS=0.000,0.000;AS_MQ=43.36,29.00;AS_MQRankSum=.,.;AS_QD=25.36,28.73;AS_ReadPosRankSum=.,.;AS_SOR=0.693,1.179;DP=66;ExcessHet=3.0103;FS=0.000;MLEAC=3,1;MLEAF=0.750,0.250;MQ=41.73;QD=30.97;RAW_MQ=114928.00;SOR=0.760	GT:AD:GQ:PL	1/2:0,6,3:99:919,177,246,420,0,500	1/1:0,20,0:74:1191,74,0,1132,93,1376
 20	10433382	.	T	C	623.88	.	AC=1;AF=0.250;AN=4;AS_BaseQRankSum=-1.900;AS_FS=0.000;AS_MQ=55.21;AS_MQRankSum=1.000;AS_QD=20.80;AS_ReadPosRankSum=0.200;AS_SOR=0.998;BaseQRankSum=-1.821e+00;DP=66;ExcessHet=3.0103;FS=0.000;MLEAC=1;MLEAF=0.250;MQ=53.72;MQRankSum=1.03;QD=20.80;RAW_MQ=86585.00;ReadPosRankSum=0.249;SOR=0.998	GT:AD:DP:GQ:PGT:PID:PL	0/1:15,15:.:99:0|1:10433382_T_C:632,0,601	0/0:36,0:36:99:.:.:0,99,1182
 20	10433391	.	T	C	622.88	.	AC=1;AF=0.250;AN=4;AS_BaseQRankSum=-0.300;AS_FS=0.000;AS_MQ=57.05;AS_MQRankSum=1.300;AS_QD=18.32;AS_ReadPosRankSum=1.500;AS_SOR=0.859;BaseQRankSum=-2.840e-01;DP=70;ExcessHet=3.0103;FS=0.000;MLEAC=1;MLEAF=0.250;MQ=55.24;MQRankSum=1.33;QD=18.32;RAW_MQ=103744.00;ReadPosRankSum=1.54;SOR=0.859	GT:AD:DP:GQ:PGT:PID:PL	0/1:18,16:.:99:0|1:10433382_T_C:631,0,673	0/0:36,0:36:99:.:.:0,99,1182
 20	10433468	.	T	C	644.88	.	AC=1;AF=0.250;AN=4;AS_BaseQRankSum=-1.400;AS_FS=2.955;AS_MQ=58.84;AS_MQRankSum=0.000;AS_QD=16.97;AS_ReadPosRankSum=1.400;AS_SOR=0.324;BaseQRankSum=-1.312e+00;DP=68;ExcessHet=3.0103;FS=2.955;MLEAC=1;MLEAF=0.250;MQ=59.00;MQRankSum=0.076;QD=16.97;RAW_MQ=132290.00;ReadPosRankSum=1.46;SOR=0.324	GT:AD:DP:GQ:PL	0/1:18,20:.:99:653,0,572	0/0:30,0:30:81:0,81,1215
-20	10433560	.	C	A	986.88	.	AC=1;AF=0.250;AN=4;AS_BaseQRankSum=3.400;AS_FS=5.027;AS_MQ=59.25;AS_MQRankSum=-1.100;AS_QD=17.62;AS_ReadPosRankSum=1.300;AS_SOR=0.129;BaseQRankSum=3.50;DP=95;ExcessHet=3.0103;FS=5.027;MLEAC=1;MLEAF=0.250;MQ=59.67;MQRankSum=-1.078e+00;QD=17.62;RAW_MQ=199369.00;ReadPosRankSum=1.38;SOR=0.129	GT:AD:DP:GQ:PGT:PID:PL	0/1:31,25:.:99:0|1:10433560_C_A:995,0,3032	0/0:39,0:39:99:.:.:0,99,1485
+20	10433560	.	C	A	986.88	.	AC=1;AF=0.250;AN=4;AS_BaseQRankSum=3.400;AS_FS=5.027;AS_MQ=59.25;AS_MQRankSum=-1.100;AS_QD=17.63;AS_ReadPosRankSum=1.300;AS_SOR=0.129;BaseQRankSum=3.50;DP=95;ExcessHet=3.0103;FS=5.027;MLEAC=1;MLEAF=0.250;MQ=59.67;MQRankSum=-1.078e+00;QD=17.62;RAW_MQ=199369.00;ReadPosRankSum=1.38;SOR=0.129	GT:AD:DP:GQ:PGT:PID:PL	0/1:31,25:.:99:0|1:10433560_C_A:995,0,3032	0/0:39,0:39:99:.:.:0,99,1485
```

Comment:
I'm assuming you've noticed the fact that the AS_QD rounding seems to have changed and that you don't care about that.

Reply:
I think it has to do with the switch from log10 space to Phred space, but really at the second decimal place it's not a big deal.